packages/postgres-database/src/simcore_postgres_database/models/projects_to_folders.py

```diff
@@ -0,0 +1,49 @@
+import sqlalchemy as sa
+
+from ._common import (
+    column_created_datetime,
+    column_modified_datetime,
+    register_modified_datetime_auto_update_trigger,
+)
+from .base import metadata
+from .folders_v2 import folders_v2
+
+projects_to_folders = sa.Table(
+    "projects_to_folders",
+    metadata,
+    sa.Column(
+        "folder_id",
+        sa.BigInteger,
+        sa.ForeignKey(
+            folders_v2.c.folder_id,
+            name="fk_projects_to_folders_to_folders_id",
+            onupdate="CASCADE",
+            ondelete="CASCADE",
+        ),
+    ),
+    sa.Column(
+        "project_uuid",
```

Comment:
don't you need a constraint to avoid having the same project in more than one folder?

Reply:
added: `sa.UniqueConstraint("project_uuid", "folder_id", "user_id"),`